client/settings/fraud-protection/index.tsx

```diff
@@ -13,20 +13,22 @@ import { ProtectionLevels } from './components';
 import './style.scss';
 import FraudProtectionTour from './tour';
 
-const FraudProtection = () => {
+const FraudProtection: React.FC = () => {
 	return (
 		<>
 			<Card className="fraud-protection">
 				<CardBody>
-					<h4>
-						<span id="fraud-protection-card-title">
-							{ __(
-								'Set your payment risk level',
-								'woocommerce-payments'
-							) }
-						</span>
-					</h4>
-					<ProtectionLevels />
+					<>
```

Comment:
We had added it before because the `<h4>` and the `<ProtectionLevels />` were at the same level and TS was complaining about it. I've removed it in [82ce8c2](https://github.com/Automattic/woocommerce-payments/pull/6032/commits/82ce8c264520073658396aed9a8ddcd4edf394ff). TS wasn't complaining about it any more. 🤷

Reply:
JSX expressions must have a single parent, but usually components' children have `ReactNode[]` type, so they don't need a Fragment.